invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -29,15 +29,20 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
   );
   const publicationDate = _get(
     result,
-    "ui.publication_date_l10n",
+    "metadata.publication_date_l10n",
```

Comment:
Minor: ``publication_date_l10n`` should be in ``ui.publication_date_l10n``. There's something wrong if it is not.